definitions/output/reports/tech_crux.js

```diff
@@ -270,14 +268,14 @@ lab_metrics AS (
 origins_summary AS (
   SELECT
     geo,
-    client,
+    l.client,
     rank,
     technology,
     version,
-    COUNT(DISTINCT root_page) AS origins
-  FROM lab_metrics
-  INNER JOIN crux
-  USING (client, root_page)
+    COUNT(DISTINCT l.root_page) AS origins
+  FROM lab_metrics l
+  INNER JOIN crux c
+  ON (l.client = IFNULL(c.client, l.client) AND l.root_page = c.root_page)
```

Comment:
@tunetheweb I think after our `device` to `client` mapping in `crux` CTE, we will not have NULL values anymore. So not sure if this is required.

Reply:
It is required, as per above.